.gitignore

```diff
@@ -8,4 +8,4 @@ dist/
 build/
 docs/
 .DS_Store
-package-lock.json
\ No newline at end of file
+lerna-debug.log
```

Comment:
don't you still need package-lock?

Reply:
I have #196 to add it in so did it as part of this refactor.  Previously when I added the .npmrc it ensured the use of private registries for local dev had no impact on the registry in the lock so I'm comfortable adding it now.